### PR TITLE
Added a leaderboard and timer for the typerace command

### PR DESCRIPTION
**Pull request by Ceres445**
using the already existing `timer` command for the timer. You could remove that if you don’t think its necessary. The results are posted at the end of 60 seconds after the quote was sent. Only top 10 users are sent. The top 3 get a flashy medal.

### CODE REVIEW

**Comment by Cordila** [comment]
Is it possible to make it so the first person is displayed as soon as he types it and the rest come in the leaderboard? Because I feel it would seem weird for the first person to type in 10s and wait 60s for all the results (Esp if he was the only one)

**Comment by Ceres445** [comment]
Ok, do you want me to edit the leaderboard message every time someone completes or should it post a new leaderboard message>

**Comment by Cordila** [comment]
Posting a new leaderboard seems perfect

File: typeracer/typeracer.py [hunk 1]
```diff
@@ -136,69 +154,85 @@ async def typeracer(self, ctx):
...
 
-        
     @commands.command(aliases=["tr"])
     @commands.cooldown(1, 10, commands.BucketType.guild)
     @commands.max_concurrency(1, commands.BucketType.channel)
     async def typerace(self, ctx, member: discord.Member = None):
-        check = await self.coll.find_one({"channel": ctx.channel.id})
-        if not check:
-            return await ctx.reply("You are not allowed to use that here", delete_after=4)
+        # check = await self.coll.find_one({"channel": ctx.channel.id})
```
Comment (Cordila) [inline typeracer/typeracer.py:218]: Why is this commented?

Comment (Ceres445) [reply typeracer/typeracer.py:218]: I had commented it out for testing, will revert it

**Pull request merged**